packages/main/src/Menu.ts

```diff
@@ -690,6 +591,7 @@ class Menu extends UI5Element {
 
 	_afterPopoverOpen() {
 		this.open = true;
+		this.items[0]?.focus();
```

Comment:
Do we need focus on mobile device? Because there is such.

Reply:
That is an old thread. Not sure if we've came to a conclusion to what we do in such situation on a framework level.
The easiest way to fix this is to add a conditional for a phone device. I suggest to leave that for a later time as there are other places where we place focus the same way.